soaks/bin/soak_one.sh

```diff
@@ -103,13 +92,28 @@ if [[ "$(docker images -q "$IMAGE" 2> /dev/null)" == "" ]]; then
   fi
 fi
 
-./run_experiment.sh --capture-dir "${CAPTURE_DIR}" \
-                    --variant "${VARIANT}" \
-                    --image "${IMAGE}" \
-                    --soak "${SOAK_NAME}" \
-                    --cpus "${SOAK_CPUS}" \
-                    --memory "${SOAK_MEMORY}" \
-                    --vector-cpus "${VECTOR_CPUS}" \
-                    --warmup-seconds "${WARMUP_SECONDS}"
+for ((idx=0; idx <= REPLICAS ; idx++))
+do
+    SOAK_CAPTURE_DIR="${CAPTURE_DIR}/${SOAK_NAME}/${VARIANT}/${idx}"
+    SOAK_CAPTURE_FILE="${SOAK_CAPTURE_DIR}/${VARIANT}.captures"
+
+    pushd "${SOAK_ROOT}/tests/${SOAK_NAME}/" > /dev/null
+    echo "[${VARIANT}] Captures will be recorded into ${SOAK_CAPTURE_DIR}"
+    mkdir -p "${SOAK_CAPTURE_DIR}"
+    touch "${SOAK_CAPTURE_FILE}"
+    # shellcheck disable=SC2140
+    DOCKER_BUILDKIT=1 docker run --env RUST_LOG="info" --env RUST_BACKTRACE="1" \
+                   --mount type=bind,source="${SOAK_ROOT}/tests/${SOAK_NAME}/lading.yaml",target="/etc/lading/lading.yaml",readonly \
+                   --mount type=bind,source="${SOAK_ROOT}/tests/${SOAK_NAME}/vector.toml",target="/etc/vector/vector.toml",readonly \
+                   --mount type=bind,source="${SOAK_ROOT}/tests/${SOAK_NAME}/data",target="/data",readonly \
+                   --mount type=bind,source="${SOAK_CAPTURE_DIR}",target="/tmp/captures" \
+                   --user "$(id -u):$(id -g)" \
+                   "${IMAGE}" \
+                   --config-path "/etc/lading/lading.yaml" \
+                   --global-labels "variant=${VARIANT},target=vector,experiment=${SOAK_NAME}" \
+                   --target-env-vars "VECTOR_THREADS=${VECTOR_CPUS}" \
```

Comment:
Not specific to this PR, and also not a big deal, but I'd find it easier to follow if these flags matched the config file keys. That is, this flag would be `--target-environment-variables` to match `target.environment_variables`. I might even make it literally `--target.environment_variables` to make it even more explicit.

Reply:
It's a fair point. I'll see if I can alias the arguments so both long and truncated are possible.

Reply:
Today `lading` uses [argh](https://lib.rs/crates/argh). No aliasing and the dot suggestion also isn't feasible. I'd be willing to cut a new release with updated CLI args if we can pull off what you're suggesting with clap.

Reply:
Addressed here: https://github.com/blt/lading/pull/150